<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authentification</class>
 <widget class="QDialog" name="Authentification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
* {
    
	 /*border: 1px solid rgb(0, 0, 0); */
	 color: rgb(74, 74, 74);
    font-family: 'Segoe UI', Arial, sans-serif;
}

#Authentification,#widget,#page,#page_2{

		 background-color: rgb(248, 249, 250);
        border: none;

}

#zone_message_erreur_page_creer_compte,#zone_message_erreur_page_se_connecter{

		
	font: italic 8pt &quot;Segoe UI&quot;;

	
	color: rgb(186, 2, 45);

}


#btn_se_connecter_zone_creer_compte,#btn_creer_compte_zone_connexion{


    background-color: rgb(245, 247, 250);
    border: 1px solid rgb(214, 221, 235);
    color: rgb(74, 74, 74);
    border-radius: 4px;
    padding: 6px 10px;

}



#btn_se_connecter_zone_creer_compte:hover , #btn_creer_compte_zone_connexion:hover{

   background-color: rgb(41, 98, 255);
	
	color: rgb(255, 255, 255);
}


#btn_mot_de_passe_oublier{

		border:none ;
		background:transparent ;
		color:rgb(41, 98, 255);

	
	font: 700 8pt &quot;Segoe UI&quot;;


}

#btn_mot_de_passe_oublier:hover{

		color:rgb(41, 98, 255,0.2);

}



/* ===============================================
   CONTENEURS ET PAGES PRINCIPALES
   =============================================== */



#carte_connexion , #carte_creer_compte {
    background-color: rgb(255, 255, 255);
    border-radius: 4px;
    border: 1px solid rgb(214, 221, 235);
}



/* ===============================================
   INPUTS DE BASE
   =============================================== */


QLineEdit, QTextEdit {
    background-color: rgb(245, 247, 250);
    border: 1px solid #D6DDEB;
    border-radius: 4px;
    padding: 6px;
    color: #4A4A4A;
}

QLineEdit:hover, QTextEdit:hover {
    border: 1px solid #2962FF;
    background-color: rgb(250, 251, 253);
}

QLineEdit:focus, QTextEdit:focus, QComboBox:focus, QDoubleSpinBox:focus {
    border: 1px solid #2962FF;
    outline: none;
}


/* ===============================================
   BOUTONS STANDARDS
   =============================================== */

QPushButton {
    background-color: rgb(41, 98, 255);
    padding: 6px 12px;
    border-radius: 4px;
    border: none;
    color: rgb(255, 255, 255);
    font-weight: 500;
    min-height: 18px;
}

QPushButton:hover {
    background-color: rgb(26, 84, 221);
}

QPushButton:pressed {
    background-color: rgb(21, 74, 201);
}

QToolButton {
    background: transparent;
    border: none;
    outline: none;
}


/* ===============================================
   SÉPARATEURS
   =============================================== */

Line {
    background-color: rgb(37, 38, 40);
    border-radius: 1px;
    border: 1px solid rgb(37, 38, 40);
    width: 1px;
}

QGroupBox {
    border: none;
    border-radius: 4px;
    margin-top: 6px;
    padding: 8px 6px 6px 6px;
    background: rgba(248, 249, 250, 0.8);
    font-weight: 500;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 8px;
    padding: 0 6px;
    color: #495057;
    font-weight: 600;
    font-size: 13px;
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>491</width>
       <height>441</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="minimumSize">
         <size>
          <width>325</width>
          <height>350</height>
         </size>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>400</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QWidget" name="carte_connexion" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>2</number>
                </property>
                <property name="leftMargin">
                 <number>10</number>
                </property>
                <property name="topMargin">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>10</number>
                </property>
                <property name="bottomMargin">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget_3" native="true">
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; color:#4387ca;&quot;&gt;My Bank .&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignmentFlag::AlignCenter</set>
                     </property>
                     <property name="textInteractionFlags">
                      <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>Connectez-vous à votre espace personnel .</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignmentFlag::AlignCenter</set>
                     </property>
                     <property name="textInteractionFlags">
                      <set>Qt::TextInteractionFlag::LinksAccessibleByKeyboard|Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextBrowserInteraction|Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>Adresse e-mail</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QLineEdit" name="lineEdit_email_connexion"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_2">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>Mot  de passe</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <widget class="QLineEdit" name="lineEdit_mot_de_passe_connexion">
                     <property name="echoMode">
                      <enum>QLineEdit::EchoMode::Password</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="zone_message_erreur_page_se_connecter">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignmentFlag::AlignLeft">
                 <widget class="QPushButton" name="btn_mot_de_passe_oublier">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>Mot  de passe oublié ?</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_se_connecter_zone_connexion">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>Se connecter</string>
                  </property>
                  <property name="icon">
                   <iconset resource="icons.qrc">
                    <normaloff>:/icon_blanc/log-in.svg</normaloff>:/icon_blanc/log-in.svg</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignTop">
              <widget class="QWidget" name="widget_4" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                 <number>25</number>
                </property>
                <item alignment="Qt::AlignmentFlag::AlignVCenter">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Vous n'avez pas de compte ?</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignCenter</set>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::TextInteractionFlag::LinksAccessibleByKeyboard|Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextBrowserInteraction|Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_creer_compte_zone_connexion">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>créer un compte</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
           <widget class="QWidget" name="widget_5" native="true">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>400</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QWidget" name="carte_creer_compte" native="true">
               <property name="minimumSize">
                <size>
                 <width>311</width>
                 <height>291</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <property name="spacing">
                 <number>2</number>
                </property>
                <property name="leftMargin">
                 <number>10</number>
                </property>
                <property name="topMargin">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>10</number>
                </property>
                <property name="bottomMargin">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget_6" native="true">
                  <layout class="QVBoxLayout" name="verticalLayout_10">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; color:#4387ca;&quot;&gt;My Bank .&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignmentFlag::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>Rejoignez Banque Moderne dès aujourd'hui.</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignmentFlag::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_3">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>Nom complet</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_11">
                   <item>
                    <widget class="QLineEdit" name="lineEdit_nom_complet_zone_creer_compte"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_4">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>Adresse e-mail</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_12">
                   <item>
                    <widget class="QLineEdit" name="lineEdit_email_zone_creer_compte"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_5">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>Mot de passe </string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_15">
                   <item>
                    <widget class="QLineEdit" name="lineEdit_mot_de_passe_zone_creer_compte">
                     <property name="echoMode">
                      <enum>QLineEdit::EchoMode::Password</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_6">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>Confirmer le mot de passe </string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_16">
                   <item>
                    <widget class="QLineEdit" name="lineEdit_confirme_mot_de_passe_zone_creer_compte">
                     <property name="echoMode">
                      <enum>QLineEdit::EchoMode::Password</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="zone_message_erreur_page_creer_compte">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignmentFlag::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_creer_compte_zone_creer_compte">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>Créer mon compte</string>
                  </property>
                  <property name="icon">
                   <iconset resource="icons.qrc">
                    <normaloff>:/icon_blanc/user-plus.svg</normaloff>:/icon_blanc/user-plus.svg</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_7" native="true">
               <property name="minimumSize">
                <size>
                 <width>311</width>
                 <height>71</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_13">
                <property name="spacing">
                 <number>25</number>
                </property>
                <item alignment="Qt::AlignmentFlag::AlignBottom">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Vous avez déjà un compte ?</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_se_connecter_zone_creer_compte">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>Se connecter</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>79</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
